mono/mini/mini-amd64.h

```diff
@@ -210,16 +210,16 @@ typedef struct MonoCompileArch {
 	gint32 sp_fp_offset;
 	guint32 saved_iregs;
 	gboolean omit_fp, omit_fp_computed;
-	gpointer cinfo;
+	CallInfo *cinfo;
 	gint32 async_point_count;
-	gpointer vret_addr_loc;
+	MonoInst *vret_addr_loc;
+	MonoInst *seq_point_info_var;
+	MonoInst *ss_tramp_var;
+	MonoInst *bp_tramp_var;
+	MonoInst *lmf_var;
 #ifdef HOST_WIN32
 	struct _UNWIND_INFO* unwindinfo;
```

Comment:
Also moved the "variable" part to the end, so more things are more the same, i.e. the offsets of the common parts are the same on all platforms.